source/endpoints/urls.md.erb

```diff
@@ -0,0 +1,755 @@
+# URLs
+
+URLs represent a site or domain for which a set of keywords are being tracked.
+URLs can also contain backlinks, and are used to perform site audits on.
+Althought these two last functionalities have their own endpoint, but their
+specific settings are configured with the URL endpoint.
+
+<aside class="notice">
+  URLs belong to a URL group, having at least one group is the prerequisite for
+  adding URLs, keywords and other resources.
+</aside>
+
+## URL data
+
+               Field                     | Description
+---------------------------------------- | -----------
+id                                       | URL ID
+url_type                                 | URL engine type (deprecated)
+url                                      | URL
+last_processed_at                        | time when this
```

Comment:
Missing description. Also let's capitalize all descriptions because it's also capitalized in other places.